### PR TITLE
Add Keep link as post processing method

### PR DESCRIPTION
**Pull request by jee-r**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Hi,
I hope I didn't make too many mistakes, the python workflow is very new to me.
Please don't hesitate to tell me what I did wrong if I did, I'd like to learn how to do things right thank you.

Related to  this issue https://github.com/pymedusa/Medusa/issues/6654

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,5 +14,6 @@
   },
   "devDependencies": {
     "dredd": "12.2.1"
-  }
+  },
+  "version": "0.0.0"
```
Comment (medariox) [inline package.json:18]: Please remove the version

**Comment by p0psicles** [comment]
An entry in the changelog.md would be nice

**Comment by jee-r** [comment]
You'r right @p0psicles as well as updating the wiki but i don't know how edit the wiki

Should i add an entry in changelog.md under the `Unreleased` section ? like this :
```
## Unreleased

#### New Features
-  Add Keep link as post processing method ([6654](https://github.com/pymedusa/Medusa/issues/6654))

#### Improvements

#### Fixes

-----

## 0.3.15 (2020-04-13)

... etc
```

Btw @medariox is my last commit good ?

**Comment by p0psicles** [comment]
Changelog is good! 
You can PR the wiki. 
I think the PR looks good. But @medariox is more into that part of the code

**Comment by p0psicles** [comment]
No need to merge develop. 
Only required when there are conflicts

**Comment by p0psicles** [comment]
We link the number of the pr in the change log. Not the issue

**Comment by medariox** [comment]
Awesome work, thanks! A Wiki entry is highly appreciated.

**Pull request merged**